### ISSUE TITLE
Unable to remove games

### ISSUE THREAD

**Issue by gapmeister11** [lutris/lutris#527]
Using version 4.5, up to date Arch Linux.

I can't remove games from my library. Every time I restart Lutris, they come back.

**Comment by daniel-j** [comment]
That would be games you have installed on your system. Try remove them with your package manager.

Lutris looks for games in /usr/share/applications/ and ~/.local/share/applications/

Lutris auto-imports them when it starts, right now there is no user-configurable blacklist.

**Comment by Lutgolhein** [comment]
Hello Team...

This is just a feedback regarding this new feature implemented in Lutris v0.4.5 - "_Scan for games before loading gui_".

Same problem @gapmeister11 above. 

Had a nice conversation with djazz (@daniel-j) in IRC about it :)

The problem is that I have duplicated games in my list, those that I added previously and, as @gapmeister11mentioned above, at this stage users are not able to delete them (apart from the workarounds described.)

Can this be optional in the next coming versions... or better yet:

Could be possible to have this feature implemented but with the possiblity of removing games if desired ?

<3 Lutris.
